api/client/webclient/webconfig.go

```diff
@@ -61,6 +61,8 @@ type WebConfig struct {
 	// The web UI for dashboards provides functionality for downloading self-hosted licenses and
 	// Teleport Enterprise binaries.
 	IsDashboard bool `json:"isDashboard,omitempty"`
+	// IsUsageBasedBilling determines if the user subscription is usage-based (pay-as-you-go).
```

Comment:
to make it clear it's a cloud related flag?
```suggestion
	// IsUsageBasedBilling determines if the cloud user subscription is usage-based (pay-as-you-go).
```